core/src/main/java/org/apache/calcite/util/format/FormatElementEnum.java

```diff
@@ -41,6 +41,13 @@
  * @see FormatModels#DEFAULT
  */
 public enum FormatElementEnum implements FormatElement {
+  CC("cc", "century (2 digits) (the twenty-first century starts on 2001-01-01)") {
```

Comment:
I think the commit message should probably be more descriptive, I wasn't sure what `Support date format` meant until I reviewed the PR.

Reply:
+1

I won't even review a PR until the jira case has a good description of its goals. Sounds pedantic but it's very pragmatic.

This jira case also needs to say which functions these date formats are supported in. Changes to FormatElementEnum are not useful unless they are surfaced in functions.